src/ice_model.F90

```diff
@@ -1274,7 +1274,7 @@ subroutine set_fast_ocean_sfc_properties( Atmos_boundary, Ice, IST, Rad, FIA, &
   ! timestep.  However, it is safe (if wasteful) to call it more frequently.
   if (Rad%do_sun_angle_for_alb) then
     call set_ocean_albedo_from_astronomy(Ice, G, Time_start, Time_end)
-  elseif (coszen_changed) then
+  elseif (coszen_changed .or. Ice%sCS%specified_ice) then
```

Comment:
Something about this mod is troublesome. If this is to change anything at all, it must be that coszen_change is false but Ice%albedo(k=1) is changing inside set_ocean_albedo_from_coszen. Yet  Ice%albedo(k=1) depends only on coszen and should not change if coszen does not!
One way out of this is if Rad%coszen_nextrad has changed since the previous time this sub was called but is still equal to Rad%coszen_lastrad. One way to test this is to set   Rad%coszen_lastrad = Rad%coszen_nextrad before line 1258 above.